src/Mod/Material/Resources/Materials/Standard/Thermoplast/PMMA-Generic.FCMat

```diff
@@ -0,0 +1,26 @@
+---
+General:
+  UUID: "476caec9-e24a-4af1-bec0-96feb0d29d5b"
+  Name: "PMMA-Generic"
+  Author: "FreeCAD"
+  License: "CC-BY-SA-4.0"
+  Description: "Poly(methyl methacrylate) (PMMA) is a synthetic polymer derived from methyl methacrylate. It is a transparent thermoplastic, used as an engineering plastic. If you need a more precise material definition use the more specialised POM-blend cards."
```

Comment:
You probably don't mean "POM-blend" here since it's an unrelated plastic.

Reply:
Argh, not having line wrapping enabled tripped me up this time... Thanks for catching this...